src/utils/messages.js

```diff
@@ -46,3 +47,24 @@ export function displayErrorMessageWithSnackbar({
         style: { whiteSpace: 'pre-line' },
     });
 }
+
+export const useSnackbarMessage = () => {
+    const intlRef = useIntlRef();
+    const { enqueueSnackbar } = useSnackbar();
+    let message;
+    message = useCallback(
+        (msg, headerMessage, headerValues) =>
```

Comment:
Rename `headerMessage` by `headerMessageId`

Reply:
Maybe `msgId` as well ? and keep interntionalization process in here ?

Reply:
msg is not an identifier but a message

Reply:
done for headerMessage